### ISSUE TITLE
Hide identity provider inside account console

### ISSUE THREAD

**Issue by mnaser** [keycloak/keycloak#13206]
### Description

At the moment, an identity provider can be hidden from login, however, users can still see all the identity providers of the realm by going into the account console and going to "Linked Accounts".

### Discussion

https://github.com/keycloak/keycloak/discussions/8839

### Motivation

This can be a problematic issue for a cloud-y IDP that allows multiple different tenants/users to login and view all their information from there.  In our case, we'd rather users only see social login to link there and their primary login method

### Details

I can try and pick this up but I would just need some guidance.

**Comment by kodebach** [comment]
We're also interested in this and currently use a custom theme to hide the IdPs.

**Comment by reda-alaoui** [comment]
Isn’t that a duplicate of https://github.com/keycloak/keycloak/issues/19732 ?

**Comment by kodebach** [comment]
You're right, thanks for the hint. Google lead me to this issue and the test instance I used to check if this has changed, was still on v26.3.2 so it didn't show the option. I guess we can update our themes now.
